MonumentAddons.cs

```diff
@@ -4392,6 +4392,54 @@ protected virtual void PostEntitySpawn()
                     candle.SetFlag(BaseEntity.Flags.Busy, true);
                 }
 
+                var fogMachine = Entity as FogMachine;
+                if (fogMachine != null)
+                {
+                    fogMachine.SetFlag(BaseEntity.Flags.On, true);
+
+                    PluginInstance.timer.Every(fogMachine.fogLength - 1f, () =>
+                    {
+                        fogMachine.SetFlag(BaseEntity.Flags.Reserved6, b: true);
```

Comment:
I suggest we remove `b: ` since I don't see it adding much value. Also in a few other places.

I also suggest using the `FogMachine` static flags for readability, such as `FogMachine.Emitting`. These don't show up in the decompiler because the compiler would have inlined them as they are constants.

Reply:
`b:` removed and noted that I should read whatever i copy from decompiler :)

Also I replaced that timer with repeating invoke